css/main.css

```diff
@@ -27,7 +27,8 @@ header {
   color: white;
 }
 
-#desktop-nav, #desktop-nav ul {
+#desktop-nav,
+#desktop-nav ul {
   display: none;
```

Comment:
[OPTIONAL]
- [ ] Could you please🙏🙏 take into consideration implementing a media query that ensures the mobile menu appears on the screen up to 768px? 
- [ ] Adhering to this guideline will help optimize the user experience for mobile devices🤔🤔